pkgs/by-name/ph/phase/update.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq nix-prefetch common-updater-scripts nix coreutils
+# shellcheck shell=bash
+set - euo pipefail
+
+  # Fetch the latest Phase CLI version from GitHub https://github.com/phasehq/cli
+  REPO="phasehq/cli"
+RELEASE_TAG=$(curl -s "https://api.github.com/repos/$REPO/releases/latest" | jq -r .tag_name)
+VERSION=${RELEASE_TAG#v}
+
+echo "Latest Phase CLI release: $VERSION"
+
+# Prepare x86 and arm64 targets for Linux and MacOS
+ARCHS=(
+"x86_64-linux:linux_amd64"
+"x86_64-darwin:macos_amd64"
+"aarch64-linux:linux_arm64"
+"aarch64-darwin:macos_arm64"
```

Comment:
You could probably make this an associative array
```suggestion
  "x86_64-linux:linux_amd64"
  "x86_64-darwin:macos_amd64"
  "aarch64-linux:linux_arm64"
  "aarch64-darwin:macos_arm64"
```